aws/resource_aws_lb_ssl_negotiation_policy_test.go

```diff
@@ -56,7 +56,7 @@ func TestAccAWSLBSSLNegotiationPolicy_disappears(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckLBSSLNegotiationPolicy(elbResourceName, resourceName),
 					testAccCheckAWSELBExists(elbResourceName, &loadBalancer),
-					testAccCheckAWSELBDisappears(&loadBalancer),
+					testAccCheckResourceDisappears(testAccProvider, resourceAwsElb(), resourceName),
```

Comment:
This is currently returning test failures:

```
--- FAIL: TestAccAWSLBSSLNegotiationPolicy_disappears (58.71s)
testing.go:684: Step 0 error: Check failed: Check 3/3 error: Error deleting ELB: ValidationError: LoadBalancer name cannot be longer than 32 characters
```

I think it needs to be switched to the `aws_elb.test` resource name to match the original test:

```suggestion
					testAccCheckResourceDisappears(testAccProvider, resourceAwsElb(), elbResourceName),
```

Reply:
@bflad, thanks for catching this, done,